src/Tizen.NUI/src/public/Utility/Palette.cs

```diff
@@ -141,7 +143,6 @@ public static void GenerateAsync(PixelBuffer pixelBuffer, PaletteGeneratedEventH
         /// <param name="region">A rectangle used for region.</param>    
         /// <param name="paletteGeneratedEventHandler">A method will be called with the palette when generated.</param>
         /// <exception cref="ArgumentNullException">Thrown when the argument pixelBuffer, PaletteGeneratedEventHandler is null.</exception>
-        [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
After removing this tag, should we add 'since tizen' tag at the same place?
like -> /// <since_tizen> 8 </since_tizen>